src/products/components/ProductListDatagrid/utils.ts

```diff
@@ -0,0 +1,44 @@
+import { ProductFilterKeys } from "../ProductListPage";
+
+export const getPriceClickSearchParams = (search: string): string => {
+  const params = new URLSearchParams(search);
+
+  const filtersIndices = Array.from(params.keys())
+    .map(key => {
+      if (key.length === 1 || key.length === 2) {
```

Comment:
suggestion: could you give comment what is this case, when key has length 1 or 2, what does it mean?

Reply:
why not using function for it with the proper name that will indicate what's happening here?